shared/settings/routes.native.tsx

```diff
@@ -15,7 +15,7 @@ export const newRoutes = {
   [Constants.deleteMeTab]: {getScreen: () => require('./delete/container').default},
   [Constants.feedbackTab]: {getScreen: () => require('./feedback/container').default},
   [Constants.invitationsTab]: {getScreen: () => require('./invites/container').default},
-  [Constants.landingTab]: {getScreen: () => require('./about-container').default},
+  [Constants.accountTab]: {getScreen: () => require('./about-container').default},
```

Comment:
I don't see a way to get here on mobile?

Reply:
Hmm, I seem to have blind find-replaced here instead of actually changing the appropriate thing. Now hooked up properly.